[FEATURE] Ajout support group et permission de Metabase (PIX-8495) (#3)
1024pix/steampipe-plugin-metabase: docs/tables/metabase_permission_download.md

# Table: metabase_permission_download

> List download permission of Metabase for getting download data from database.

Reviewer: ```suggestion
List download permission of Metabase to download data from database.
```
? Pas sur
